content/using/os/filesystem.md

```diff
@@ -10,132 +10,139 @@ referentially transparent namespace. An easy way to think about it is like typed
 
 The most common way to use Clay is to mount a Clay node in a Unix directory. The mounted directory is always at the root of your pier directory.
 
+For more information on Clay, see the [Overview](@/docs/arvo/clay/clay.md), and
+additional usage information at [Using Clay](@/docs/arvo/clay/using.md). 
+
 ### Quickstart
 
 This quick-start guide will walk you through some common commands. Follow along
 using your Dojo. When you get a `>=` message after entering a command, this means
 that the command was successful.
 
+A `desk` is something like an independently revision-controlled branch of your
```

Comment:
Is there anything that we can link to about a [`desk`]? E.g. a glossary entry? If not, disregard.